Source/Application/ITMO.Dev.ASAP.Application.Contracts/Identity/Commands/ChangeUserRole.cs

```diff
@@ -0,0 +1,8 @@
+using MediatR;
+
+namespace ITMO.Dev.ASAP.Application.Contracts.Identity.Commands;
+
+internal static class ChangeUserRole
+{
+    public record struct Command(string Username, string UserRole) : IRequest;
```

Comment:
убери у себя из темплейта, надо будет потом пробежаться по контрактам и везде структуры убрать, медиатр всё равно их боксит

```suggestion
    public record Command(string Username, string UserRole) : IRequest;
```